warehouse/models/staging/transit_database/_stg_transit_database.yml

```diff
@@ -330,3 +330,45 @@ models:
       - name: link_to_formal_definition
       - name: services
       - name: dt
+  - name: stg_transit_database__issue_types
+    description: TODO
+    columns:
+      - name: id
+      - name: dataset_type
+      - name: transit_data_quality_issues
+      - name: name
+      - name: notes
+      - &dt
```

Comment:
If `dt` and `ts` appear elsewhere above, can you use these anchors for those too? (I see `dt` on line 332)

Reply:
Resolved in [ce2d2e4](https://github.com/cal-itp/data-infra/pull/2781/commits/ce2d2e4af50b81448381ff68ec3c28a31913732c)